pilot/pkg/kube/inject/inject_test.go

```diff
@@ -444,6 +449,7 @@ func newTestParams() *Params {
 		InitImage:           InitImageName(unitTestHub, unitTestTag, false),
 		ProxyImage:          ProxyImageName(unitTestHub, unitTestTag, false),
 		ImagePullPolicy:     "IfNotPresent",
+		IstioProxyVersion:   unitTestProxyVersion,
```

Comment:
If you want to add a test - add one to check env variables are propagated. This is just one particular env.